src/css/advertisement.scss

```diff
@@ -32,3 +32,29 @@
     font-size: 14px;
   }
 }
+
+.data-search-coll {
+  display: none;
+}
+
+.received-advertisements-deposits {
```

Comment:
it match the label, which is good, but it is already the code, so it should respect "everything single" and other standards
rename to `advertisement_consumer_settlement_ledger_list`